bin/pinto

```diff
@@ -279,6 +281,12 @@ when using a local repository.
 Sets the timeout (in seconds) for obtaining a lock on the repository.  The default
 is 50.  This variable only has effect when using a local repository.
 
+=item C<PINTO_GNUPGHOME>
+
+Sets the GNUPGHOME to be used when verifying upstream distribution files using
+GnuPG.  See the documentation for the C<audit> command for the rationale and
```

Comment:
The command name ought to be a link, such as `L<audit|App::Pinto::Command::audit>`